CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * Extend schema for profile relations manipulations (UIDATIMP-1471)
 * Change validation messages which were represented in code format (UIDATIMP-1473)
 * *BREAKING* bump `react` to `v18`, and dev-deps accordingly (UIDATIMP-1485)
+* Match profile: update options for Instance "Incoming records" (UIDATIMP-1501)
```

Comment:
Could you move it at the end of the list?

Reply:
Hi @artem-blazhko In Data Import we add ticket in ascending order in order to simplify finding the tickets